api/go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/ceph/ceph-csi-operator/api
+
+go 1.22.5
```

Comment:
have like this https://github.com/ceph/ceph-csi-operator/blob/1dda58616ce3c72b7903b439267ca3e46e1dbd0a/go.mod#L3-L4

Reply:
updated with this change